src/components/queue/QueueOptions/QueueOptions.tsx

```diff
@@ -49,6 +49,8 @@ const QueueOptions: FC<QueueOptionsProps> = ({queue, queueID, showCompletedTicke
     const isLongEnded = add(queue.endTime, {minutes: 30}) < new Date();
     // const isLongEnded = queue.endTime < new Date();
 
+    const lastTicketID = queue.tickets[queue.tickets.length - 1];
```

Comment:
What if the queue has no tickets? `lastTicketID` would be -1

Reply:
I believe if the queue as no tickets, the lastTicketID would be `undefined` (since this array index would be -1 and out of bounds). The undefined is passed along and displayed so that the cutoff line is at the top of the queue, which is expected behavior.